cypress.json

```diff
@@ -1 +1,10 @@
-{}
+{
+  "baseUrl": "http://localhost:8100",
+  "env": {
```

Comment:
Move the env vars to cypress.env.json, leave cypress.json for things that are going to be consistent values.

Leave baseUrl because that wouldn't not be expected to change.

Reply:
Resolved.